install.sh

```diff
@@ -25,26 +25,27 @@ fi
 # Reject crostini.
 if [[ -d /opt/google/cros-containers && "${CREW_FORCE_INSTALL}" != '1' ]]; then
   echo_error "Crostini containers are not supported by Chromebrew :/"
-  echo_info "Run 'CREW_FORCE_INSTALL=1 exec bash --init-file <(curl -Ls git.io/vddgY)' to perform install anyway"
+  echo_info "Run 'CREW_FORCE_INSTALL=1 bash <(curl -L git.io/vddgY) && . ~/.bashrc' to perform install anyway"
   exit 1
 fi
 
 # Reject non-stable Chrome OS channels.
 if [ -f /etc/lsb-release ]; then
-  if [[ ! "$(< /etc/lsb-release)" =~ CHROMEOS_RELEASE_TRACK=stable-channel$'\n' && "${CREW_FORCE_INSTALL}" != '1' ]]; then
+  export "$(grep CHROMEOS_RELEASE_CHROME_MILESTONE /etc/lsb-release)"
+
+  if ! (grep -q 'CHROMEOS_RELEASE_TRACK=stable-channel' /etc/lsb-release || [[ "${CREW_FORCE_INSTALL}" == '1' ]]); then
```

Comment:
Is there a way to avoid grepping `/etc/lsb-release` twice?

Reply:
We can use `mapfile` to parse the whole file first, but I am not familiar with it 😅